client/my-sites/plans/trials/trial-banner/use-banner-subtitle.ts

```diff
@@ -70,6 +71,31 @@ export default function useBannerSubtitle(
 				);
 			}
 		}
+		let entrepreneurTrialSubtitle;
+		if ( isEntrepreneurTrial ) {
+			if ( trialDaysLeftToDisplay < 1 ) {
+				entrepreneurTrialSubtitle = translate(
+					'Your free trial ends today. Add payment method by %(expirationdate)s to continue selling your products and access all the power.',
+					{
+						args: {
+							expirationdate: readableExpirationDate as string,
+						},
+					}
+				);
+			} else {
+				entrepreneurTrialSubtitle = translate(
+					'Your free trial will end in %(daysLeft)d day. Add payment method by %(expirationdate)s to continue selling your products and access all the power.',
```

Comment:
Would it be possible to add plural here? (1 day, 2 days, etc.) Some languages have multiple plurals, so this would be helpful in getting an accurate translation.

Reply:
Nice catch! This is being addressed in https://github.com/Automattic/wp-calypso/pull/89878.